molr-mole-core/src/main/java/io/molr/mole/core/tree/TreeMissionExecutor.java

```diff
@@ -108,5 +124,27 @@ public void instructRoot(StrandCommand command) {
         instruct(strandFactory.rootStrand(), command);
     }
 
-
+    @Override
+    public void instructBlock(String blockId, BlockCommand command) {
```

Comment:
Looks like this implementation would also be simpler if using the block object and not the id, or?